ReactiveCocoa.xcodeproj/xcshareddata/xcschemes/ReactiveSwift-Mac.xcscheme

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "0730"
+   version = "1.3">
+   <BuildAction
+      parallelizeBuildables = "YES"
+      buildImplicitDependencies = "YES">
```

Comment:
Can you please set to `NO` for these to settings as same as [ReactiveCocoa-Mac.xcscheme](https://github.com/ReactiveCocoa/ReactiveCocoa/blob/v4.2.1/ReactiveCocoa.xcodeproj/xcshareddata/xcschemes/ReactiveCocoa-Mac.xcscheme#L6-L7) and other ReaectiveCocoa schemes?